kubernetes/docs/try-with-minikube.md

```diff
@@ -0,0 +1,60 @@
+# Try with Minikube
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [Kubernetes](https://kubernetes.io/)
+- [Minikube](https://github.com/kubernetes/minikube)
+- [Helm](https://helm.sh/)
+
+Please follow
+[instructions](https://kubernetes.io/docs/tasks/tools/install-minikube/) to
+install Minikube.
+
+## Create/start a minikube cluster
+
+```sh
+minikube start -p shopping-cluster
+```
+
+## Build docker images for shopping and recommender for the minikube cluster
+
+```sh
+eval $(minikube -p shopping-cluster docker-env)
+npm run docker:build
+```
+
```

Comment:
This essentially calls

`./bin/build-images.sh`

Reply:
Yes, but we prefer `npm` scripts.